packages/openapi/openapi.0.6.0/opam

```diff
@@ -0,0 +1,42 @@
+opam-version: "2.0"
+synopsis: "Openapi documentation generation for Opium"
+description:
+  "Shim layer around Opium.App (https://github.com/rgrinberg/opium/) for producing Openapi documentation pages"
+maintainer: ["J. Aaron Pendergrass <james.pendergrass@jhuapl.edu>"]
+authors: ["J. Aaron Pendergrass <james.pendergrass@jhuapl.edu>"]
+license: "BSD-3-Clause"
+homepage: "https://github.com/jhuapl-saralab/openapi-ocaml"
+bug-reports: "https://github.com/jhuapl-saralab/openapi-ocaml/issues"
+depends: [
+  "dune" {>= "2.8"}
+  "opium" {= "0.20.0"}
+  "yojson" {= "1.7.0"}
+  "core" {= "v0.14.1"}
+  "ppx_yojson_conv" {= "v0.14.0"}
+  "ppx_deriving" {= "5.2.1"}
```

Comment:
however, if you want a lock file `opam lock` might be of interest for you. But in this particular PR I don’t think this would be worthy of a release in opam-repository.

Reply:
Thanks. Makes sense. Change applied.